src/AddAggregatesPlugin.ts

```diff
@@ -239,6 +288,80 @@ const AddAggregatesPlugin: Plugin = builder => {
         },
         {}
       ),
+      ...pgIntrospectionResultsByKind.procedure.reduce(
+        (memo: GraphQLFieldConfigMap<any, any>, proc: PgProc) => {
+          /* TODO: Do fields need to be omitted? */
+          const attrIsNumberLike =
+            pgIntrospectionResultsByKind.typeById[proc.returnTypeId]
+              .category === "N";
+          if (attrIsNumberLike && pgColumnFilter(proc, build, context)) {
+            const computedColumnDetails = getComputedColumnDetails(
+              build,
+              table,
+              proc
+            );
+            if (computedColumnDetails) {
+              const fieldName = inflection.computedColumn(
+                computedColumnDetails.pseudoColumnName,
+                proc,
+                table
+              );
+              return build.extend(memo, {
+                [fieldName]: pgField(
+                  build,
+                  fieldWithHooks,
+                  fieldName,
+                  ({ addDataGenerator }: any) => {
+                    addDataGenerator((parsedResolveInfoFragment: any) => {
+                      return {
+                        pgQuery: (queryBuilder: QueryBuilder) => {
+                          // Note this expression is just an sql fragment, so you
+                          // could add CASE statements, function calls, or whatever
+                          // you need here
+                          const expr = sql.fragment`${sql.identifier(
+                            proc.namespaceName,
+                            proc.name
+                          )}(${queryBuilder.getTableAlias()})`;
```

Comment:
^ This bit we definitely need to extract a helper for. Aggregates over computed columns that have arguments would be pretty cool.

```graphql
{
  allFoos {
    aggregates {
      sum {
        calc1: computedColumn(a: 1)
        calc2: computedColumn(a: 2)
      }
    }
  }
}
```